<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-GLPUBTPG" failures="0" tests="2" name="extentReportsInterviewPractice.ExtentReportExample" time="0.053" errors="1" timestamp="2025-08-28T17:51:00 IST" skipped="1">
  <testcase classname="extentReportsInterviewPractice.ExtentReportExample" name="startbroswer" time="0.053">
    <error message="io/reactivex/rxjava3/subjects/PublishSubject" type="java.lang.NoClassDefFoundError">
      <![CDATA[java.lang.NoClassDefFoundError: io/reactivex/rxjava3/subjects/PublishSubject
at com.aventstack.extentreports.ReactiveSubject.<init>(ReactiveSubject.java:28)
at com.aventstack.extentreports.AbstractProcessor.<init>(AbstractProcessor.java:19)
at com.aventstack.extentreports.ExtentReports.<init>(ExtentReports.java:55)
at extentReportsInterviewPractice.ExtentReportExample.startbroswer(ExtentReportExample.java:93)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:358)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
Caused by: java.lang.ClassNotFoundException: io.reactivex.rxjava3.subjects.PublishSubject
at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:528)
... 21 more
]]>
    </error>
  </testcase> <!-- startbroswer -->
  <testcase classname="extentReportsInterviewPractice.ExtentReportExample" name="GoogleTitle" time="0.000">
    <skipped/>
  </testcase> <!-- GoogleTitle -->
</testsuite> <!-- extentReportsInterviewPractice.ExtentReportExample -->
